External/SPMmodified/xASL/xASL_adm_UnzipNifti.m

```diff
@@ -45,7 +45,7 @@
 % Checks if the pathIn was given as an absolute path or if as a file-name only
 if isempty(pathstr)
 	% If a file-name only, then add the full current path to avoid ambiguity
-	pathstr = xASL_adm_UnixPath(pwd());
+	pathstr = pwd();
```

Comment:
If this is used internally in Matlab only, then `xASL_adm_UnixPath` should probably indeed go out. If there is still a `system` command somewhere using `pathstr`, then we would have to test it in WSL?

Reply:
Yes, Henk, please try to run a single test dataset on a study called "My study" with a space to verify is WSL works.